### ISSUE TITLE
Device Dectected but Not Working

### ISSUE THREAD

**Issue by Skyferia** [nefarius/ScpToolkit#387]
Issue as in image below:
![image](https://cloud.githubusercontent.com/assets/17698425/17098109/48020a74-5293-11e6-92aa-3c59d1b5d9b7.png)

Lights @ controller flashes.

Also the SCP Monitor status:
![image](https://cloud.githubusercontent.com/assets/17698425/17098131/63ac5b9e-5293-11e6-93f6-66a515e4187d.png)

What I installed:
![image](https://cloud.githubusercontent.com/assets/17698425/17098147/790ffbb2-5293-11e6-9483-46d2ec6bff43.png)

Also re-installed several times, ran_ScpToolkit Clean Wipe Utility_ prior to each install.

Controller is original PS3 DS3.

Please advice. Thank you.

**Comment by Skyferia** [comment]
Never mind, fixed it.

How: Reset controller
How to reset: Used a toothpick and pressed @ the small button in the small hole @ the back of controller

Hope this will help others.

**Comment by tosho** [comment]
Reseting the controller doesn't work for me (PS3 DS3 / Windows 10 x64). 
I tried to uninstall my bluetooth drivers and SCPToolkit and restarting a lot but the GamePad can't be found in the games and in the Control Panel > Game Controllers. SCP Monitor shows the controller to be connected (both with cabel and via bluetooth) also I can do Input Tester which shows that the controller is working but still can't be found in all my games. 
This happened just today. I had previous problem with the Bluetooth - when I restart the PC the controller can't connect via Bluetooth so I decided to Uninstall all and make clean install. I haven't be so wrong.
